da/mock/mock.go

```diff
@@ -71,6 +73,9 @@ func (m *MockDataAvailabilityLayerClient) SubmitBlock(block *types.Block) da.Res
 func (m *MockDataAvailabilityLayerClient) CheckBlockAvailability(header *types.Header) da.ResultCheckBlock {
 	hash := header.Hash()
 	_, err := m.dalcKV.Get(hash[:])
+	if errors.Is(err, badger.ErrKeyNotFound) {
```

Comment:
I'm not sure about the data I should return here.

Reply:
```suggestion
	if errors.Is(err, store.ErrKeyNotFound) {
```